akka-http-core/src/main/scala/akka/http/impl/engine/client/HttpsProxyGraphStage.scala

```diff
@@ -48,7 +49,7 @@ private final class HttpsProxyGraphStage(targetHostName: String, targetPort: Int
 
   override def shape: BidiShape[ByteString, ByteString, ByteString, ByteString] = BidiShape.apply(sslIn, bytesOut, bytesIn, sslOut)
 
-  private val connectMsg = ByteString(s"CONNECT $targetHostName:$targetPort HTTP/1.1\r\nHost: $targetHostName\r\n\r\n")
```

Comment:
Oops. Thats a mistake by my part. In my testing I made it mutable for mess with stuff than forgot to change it back.